lib/themes/dosomething/paraneue_dosomething/includes/form.inc

```diff
@@ -10,9 +10,6 @@ function paraneue_dosomething_form_alter_base(&$form, &$form_state, $form_id) {
   if(isset($form['actions']['submit'])) {
     $form['actions']['submit']['#attributes']['class'][] = 'button';
   }
-
-  // Check if this form needs Shirt Styles applied.
-  paraneue_dosomething_form_alter_shirt_options($form, $form_state, $form_id);
```

Comment:
Yep, that's a [pattern in Forge](http://forge.dosomething.org/#forms-media-option-field)! Definitely something that could be removed next time we make a big ol' version bump.